### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by gitNavV**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,19 +46,30 @@ Follow the rules, bad code will be rejected, or modified and credit denied. Yes,
   ```source myvenv/bin/activate``` <br>
   Now, you should see a (myvenv) script before cmd commands. <br>
```
Comment (markroxor) [inline README.md]: Dont be so verbose. Remove all the comments after each commamd.

File: README.md [hunk 2]
```diff
@@ -46,19 +46,30 @@ Follow the rules, bad code will be rejected, or modified and credit denied. Yes,
...
 
-* Now, do
-  ```pip install django==1.9.7```
+* Next, check updates (if any) for pip module by
```
Comment (markroxor) [inline README.md]: rm

File: README.md [hunk 3]
```diff
@@ -46,19 +46,30 @@ Follow the rules, bad code will be rejected, or modified and credit denied. Yes,
...
+  ```pip install --upgrade pip```
+
+* After that, do
```
Comment (markroxor) [inline README.md]: rm

File: README.md [hunk 4]
```diff
@@ -46,19 +46,30 @@ Follow the rules, bad code will be rejected, or modified and credit denied. Yes,
...
+  ```pip install django```
+  
+* Next, do
```
Comment (markroxor) [inline README.md]: rm

File: README.md [hunk 5]
```diff
@@ -46,19 +46,30 @@ Follow the rules, bad code will be rejected, or modified and credit denied. Yes,
...
+* Next, check updates (if any) for pip module by
+  ```pip install --upgrade pip```
+
+* After that, do
+  ```pip install django```
+  
+* Next, do 
+  ```pip install django_extensions```
+  
+* After that, do
```
Comment (markroxor) [inline README.md]: rm

File: README.md [hunk 6]
```diff
@@ -46,19 +46,30 @@ Follow the rules, bad code will be rejected, or modified and credit denied. Yes,
...
+  ```pip install Pillow```
 
 * After installation is complete, do
-  ```python manage.py makemigration mainsite```
+  ```python manage.py makemigrations mainsite```
   And
```
Comment (markroxor) [inline README.md]: rm

File: README.md [hunk 7]
```diff
@@ -46,19 +46,30 @@ Follow the rules, bad code will be rejected, or modified and credit denied. Yes,
...
+  
+* After that, do
+  ```pip install Pillow```
 
 * After installation is complete, do
-  ```python manage.py makemigration mainsite```
+  ```python manage.py makemigrations mainsite```
   And 
   ```python manage.py migrate mainsite```
   And
```
Comment (markroxor) [inline README.md]: rm

File: README.md [hunk 8]
```diff
@@ -46,19 +46,30 @@ Follow the rules, bad code will be rejected, or modified and credit denied. Yes,
...
   ```python manage.py collectstatic```
 
 * Finally, to run a virtual server, do
```
Comment (markroxor) [inline README.md]: rm

**Review by markroxor — CHANGES_REQUESTED** [review]

**Review by gitNavV — COMMENTED** [review]
Doing

**Review by markroxor — APPROVED** [review]

**Pull request merged**